src/components/navigation/header/config/HeaderNavigationConfig.tsx

```diff
@@ -60,8 +62,8 @@ export class HeaderNavigationConfig extends BaseConfig<HeaderNavigationConfigIns
   config: HeaderNavigationConfigInterface;
   homeUrl: string;
   menuHeight: string;
-  user: User | null;
-  userType: UserType;
+  user: MergedUser | null;
+  userType: UserType.Guest | MappedUserType;
```

Comment:
We also might not need to have `UserType` in header with `professional` and `private` as we use them from `MappedUserType`, maybe just having `GuestUserType` or so

Reply:
I removed the user type and replaced it by UserTypeExternal, so we just have Guest there